user-guide/Reference/DataMiner_Tools/Dataminer_Health_Check/Dataminer_Health_Check_Tool_Configuration.md

```diff
@@ -0,0 +1,161 @@
+---
+uid: DataMiner_Health_Check_Tool_Configuration
+---
+
+# Configuring the DataMiner Health Check Tool
+
+To begin receiving results from the DataMiner Health Check Tool, you will first need to [configure the test cases](#configuring-test-cases) and [configure the general report settings](#configuring-email-reports).
+
+## Configuring test cases
+
+There are two main types of test cases that can be configured, [script-based tests](#script-based-tests) and [subscription-based tests](#parameter-subscription-based-tests), each providing a different mechanism for querying data to allow for greater flexibility depending on the kind of data being tested.
+
+### Script-based tests
+
+Script-based tests use custom Automation scripts to execute and retrieve data. These tests can retrieve data from various sources, including DataMiner elements (via SLNet calls), DataMiner Agents (via DMS calls), or other external systems the DMS communicates with. Once the data is retrieved, it is evaluated against predefined thresholds to determine if the test passes or fails.
+
+Several default tests are available related to the state of the DataMiner System, but you can also **integrate new tests using Automation scripts**. For the Health Check tool to recognize the new tests, the Automation scripts must be placed in the following folder in the Automation module: *Automation Scripts* > *Health Check* > *Health Check Tests*.
+
+![Health Check tests in the Automation module](~/user-guide/images/Health_Check_Automation_Path.png)
+
+#### Adding a script-based test
+
+1. Make sure the script for the test you want to add is available in the Automation module in the folder *Automation Scripts* > *Health Check* > *Health Check Tests*.
+
+1. In DataMiner Cube, open the *Health Check Manager* element and go to the *Test Configuration* page.
+
+   This page shows a list pre-configured tests related to your DataMiner System's health. These include tests for checking the status of DMAs in the cluster, monitoring active timeouts, and more.
+
+1. Right-click the *Test Configuration* table and select *Add Test* in the context menu.
+
+   ![Add Test option](~/user-guide/images/Health_Check_Add_Test.png)
+
+1. In the pop-up window, fill in the required fields:
+
+   - *Test Type*: Make sure *Script* is selected.
+   - *Name*: Select an available script-based test.
+   - *Operator* and *Threshold*: Define the conditions for the test to determine whether it passes or fails.
+   - *Display Title*: Optionally, enter a custom name for the test to make it easily identifiable. If provided, this title will be used to identify the test in the email report; otherwise, the default test name will be displayed.
+   - *Element/DMA Filter*: Use this field to apply the test to a specific element. Enter the element/DMA you wish to target.
+   - *Display Key Filter*: Apply this filter if you want the test to consider only certain entries for the test.
```

Comment:
Which format should be used for this? Are wildcards supported?

Reply:
It should be in the format of a regular expression.